internal/model/label.go

```diff
@@ -24,3 +24,9 @@ var BetaLabel Label = Label{
 	Description: "This plugin is currently in Beta and is not recommended for use in production.",
 	URL:         "https://mattermost.com/pl/default-beta-plugins",
 }
+
+var EnterpriseLabel Label = Label{
+	Name:        "Enterprise",
+	Description: "This plugin only works on servers with an E20 license.",
+	URL:         "https://mattermost.com/pl/TODO",
```

Comment:
@aaronrothschild Should there be a link to some documentation page? Or is no link needed?

Reply:
```suggestion
	URL:         "https://mattermost.com/pricing/",
```